### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by juwara0**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [ ] #patch# - backwards-compatible bug fix
- [X] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

Closes: https://github.com/ciena-frost/ember-frost-test/issues/39

# CHANGELOG
* **Updated** travis CI configuration to remove testing since there are no tests to run
* **Removed** travis scripts that are no longer needed since we are not running test in CI
* **Updated** blueprints to install testing packages at the versions we desire
* **Added** `test-helper.js` blueprint file since we are no longer running the `ember-cli-mocha` blueprints that would have updated that file
* **Updated** the addPackagesToProject method in the blueprints to respect the `save-exact` npm flag
* **Updated** `README.md` to list all testing tools that this add-on installs via blueprints
* **Updated** version of `ember-ajax` to `^2.5.6`
* **Removed** unused `ember-cli-release` package
* **Removed** unused `ember-cli-sri` package
* **Removed** unused `ember-sinon` package
* **Removed** unused `ember-hook` package
* **Removed** unused `ember-cli-mocha` package since no tests in this addon
* **Removed** unused `ember-cli-chai` package since no tests in the addon
* **Added** `NOTES.md` file explaining the behavior/reason for implementation of this addon
* **Updated** version of `ember-test-utils` to `^8.0.0`
* **Updated** version of `ember-cli-htmlbars-inline-precompile` to `0.3.12`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,17 @@ ember install ember-frost-test
...
+* [ember-cli-mocha](https://github.com/ember-cli/ember-cli-mocha) - This installs the [Mocha](http://mochajs.org)
+ testing framework.
+* [ember-cli-chai](https://github.com/ember-cli/ember-cli-chai) - This install the [Chai](http://chaijs.com)
+ assertion library.
 * [ember-hook](https://github.com/Ticketfly/ember-hook) - This is a tool we use to create a separation between the
 DOM and our items under test.
 * [ember-sinon](https://github.com/csantero/ember-sinon) - This is our method spying/stubbing/mocking tool.
 * [ember-test-utils](https://github.com/ciena-blueplanet/ember-test-utils) - These are our test helpers that can be
 used to help test frost components.
+* [chai-jquery](https://github.com/chaijs/chai-jquery) - This is a chai extention that provides assertions for jQuery.
```
Comment (notmessenger) [inline README.md]: `extention` -> `extension`

File: README.md [hunk 2]
```diff
@@ -13,13 +13,17 @@ ember install ember-frost-test
...
+* [sinon-chai](http://chaijs.com/plugins/sinon-chai/) - This is a chai extention that provides assertions for sinon.js.
```
Comment (notmessenger) [inline README.md]: `extention` -> `extension`

File: README.md [hunk 3]
```diff
@@ -13,13 +13,17 @@ ember install ember-frost-test
...
 
-* [ember-cli-mocha](https://github.com/ember-cli/ember-cli-mocha) - This is our testing framework.
-It also includes, chaijs our assertion library.
+* [ember-cli-mocha](https://github.com/ember-cli/ember-cli-mocha) - This installs the [Mocha](http://mochajs.org)
+ testing framework.
+* [ember-cli-chai](https://github.com/ember-cli/ember-cli-chai) - This install the [Chai](http://chaijs.com)
+ assertion library.
 * [ember-hook](https://github.com/Ticketfly/ember-hook) - This is a tool we use to create a separation between the
 DOM and our items under test.
 * [ember-sinon](https://github.com/csantero/ember-sinon) - This is our method spying/stubbing/mocking tool.
```
Comment (notmessenger) [inline README.md]: Use same format as for `ember-cli-mocha` and `ember-cli-chai`.  There is no link to Sinon

File: blueprints/ember-frost-test/index.js [hunk 4]
```diff
@@ -2,90 +2,52 @@
...
-      .then(() => {
-        return this.removeLoadingOfTestHelpers()
-      })
-      .then(() => {
-        return this.addAddonsToProject(addonsToAdd)
-      })
-      .then(() => {
-        return this.addPackagesToProject(packagesToAdd)
-      })
+    return this.addPackagesToProject(packagesToAdd).catch((error) => {
```
Comment (notmessenger) [inline blueprints/ember-frost-test/index.js]: Check the chaining order of the promise methods

File: package.json [hunk 5]
```diff
@@ -28,27 +27,21 @@
   "devDependencies": {
     "bower": "^1.8.2",
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
+    "ember-ajax": "^2.5.6",
     "ember-cli": "2.12.3",
-    "ember-cli-chai": "^0.4.3",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-frost-blueprints": "^4.0.0",
```
Comment (notmessenger) [inline package.json]: Do we have any content in this repo that we would have created using the blueprints provided by this, or anticipate using such content? Do we want to keep it for consistency or remove it?  Think we might have had this conversation already...

File: package.json [hunk 6]
```diff
@@ -28,27 +27,21 @@
...
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
-    "ember-hook": "^1.4.2",
     "ember-load-initializers": "^0.6.0",
     "ember-resolver": "^2.0.3",
-    "ember-sinon": "^0.7.0",
     "ember-source": "~2.12.0",
-    "ember-test-utils": "^7.0.1",
+    "ember-test-utils": "^8.0.0",
```
Comment (notmessenger) [inline package.json]: Do we have any content in this repo that benefits from the linting? Guess there's some files...

File: package.json (hunk 5 above)
Comment (juwara0) [reply package.json]: I wanted to keep it in case we need to add some tests at some point

File: package.json (hunk 6 above)
Comment (juwara0) [reply package.json]: Yes, we need this to provide linting via `ember run test`

**Comment by notmessenger** [comment]
👍

**Comment by juwara0** [comment]
👍

**Pull request merged**